src/main/java/org/datadog/jenkins/plugins/datadog/logs/LogWriterFactory.java

```diff
@@ -15,11 +15,11 @@ public class LogWriterFactory {
     private static final String POLLING_TIMEOUT_ENV_VAR = "DD_JENKINS_LOGS_POLLING_TIMEOUT_SECONDS";
     private static final String BATCH_SIZE_LIMIT_ENV_VAR = "DD_JENKINS_LOGS_BATCH_SIZE_LIMIT";
 
-    private static final int DEFAULT_QUEUE_CAPACITY = 5_000;
-    private static final int DEFAULT_SUBMIT_TIMEOUT_SECONDS = 5;
+    private static final int DEFAULT_QUEUE_CAPACITY = 10_000;
+    private static final int DEFAULT_SUBMIT_TIMEOUT_SECONDS = 0;
```

Comment:
The opposite actually, with 0 timeout any trace/log that does not fit into the queue will be dropped immediately.
The idea is to avoid blocking the client's code and the Jenkins core. If the queue fills up, it means traces/logs are being produced faster than they're being dispatched, and a timeout will not resolve this problem, just postpone it.